lib/media/media_source_engine.js

```diff
@@ -328,7 +328,9 @@ shaka.media.MediaSourceEngine.prototype.init = async function(
         'Type negotiation should happen before MediaSourceEngine.init!');
 
     let mimeType = shaka.util.MimeUtils.getFullType(
-        stream.mimeType, stream.codecs);
+        stream.mimeType, stream.codecs)
+        // Convert mimeType to lowerCase to avoid mismatch of cases
+        .toLowerCase();
```

Comment:
I think it'd be clearer if you didn't attach the toLowerCase(); to the last line.
Like...

    // Convert mimeType to lowerCase to avoid mismatch of cases
    mimeType = mimeType.toLowerCase();

That way you don't need to have a weirdly-indented comment like that.